tests/auto_class/test_auto_classifier.py

```diff
@@ -19,8 +19,8 @@ def test_create_classifier(indico_client, testdir_file_path):
     )
     aclass.set_file_paths()
     aclass.create_classifier(verbose=False)
-    assert len(aclass.file_texts) == 2
-    assert len(aclass.file_classes) == 2
+    # assert len(aclass.file_texts) == 2
```

Comment:
I'm assuming we will start deprecating auto classifier once this goes through?

Reply:
I think we should deprecate it now

Reply:
sounds good. deprecating